### DISCUSSION TITLE
Crash casting Healing Wind

### DISCUSSION THREAD

**Discussion by Zi0nQ** [tsunamods-codes/7th-Heaven#138] [Q&A] [answered]
Buenas, comento en castellano, que me explico mejor. 
Cada vez que intento usar Ala Sanadora con Aerith, me crashea el juego. Lo he probado todo: Cambiar todas las configuraciones de gráficos, quitar todos los mods y nada. He comprobado que es solo cuando lo lanzo con el 7th Heaven en modo normal (aunque esté sin mods descargados), porque si lo lanzo sin mods (también desde el launcher de 7th Heaven), no tengo ese problema.
A ver, que puedo jugarlo sin mods o no utilizar el límite y listo, pero es una pena que me crashee con eso.

Gracias de antemano y también por el trabajazo de Julián y de todos los modders.

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hello @Zi0nQ thanks for your kind words. Any support you may require from 7th should go through 7th support channels. Feel free to join Discord and ask for help there. Cheers!
